packages/flutter/lib/src/material/slider.dart

```diff
@@ -1985,6 +1998,47 @@ class _SliderDefaultsM2 extends SliderThemeData {
 
   @override
   SliderComponentShape? get valueIndicatorShape => const RectangularSliderValueIndicatorShape();
+
+  @override
+  SliderComponentShape? get thumbShape => const RoundSliderThumbShape();
+
+  @override
+  SliderTrackShape? get trackShape => const RoundedRectSliderTrackShape();
+
+  @override
+  SliderComponentShape? get overlayShape => const RoundSliderOverlayShape();
+
+  @override
+  SliderTickMarkShape? get tickMarkShape => const RoundSliderTickMarkShape();
+
+  @override
+  Color? get valueIndicatorColor {
+    if (_sliderTheme.valueIndicatorShape is RoundedRectSliderValueIndicatorShape) {
+      return _colors.inverseSurface;
+    }
+    return _colors.primary;
+  }
+
+  @override
+  MaterialStateProperty<Size?>? get barThumbSize =>
```

Comment:
Nit: Remove `=>` when followed by a multiline block.

Reply:
Thank you!